### ISSUE TITLE
Getting Logits of Tokens

### ISSUE THREAD

**Issue by nimasadri11** [google-research/t5x#425]
In inference, I want to get the logit over vocabulary for the first generated token. How can I do that?

**Comment by nimasadri11** [comment]
@adarob  @hwchung27 @hthu @jacobaustin123 Any help on this would be appreciated : )
